lib/messages/message.js

```diff
@@ -9,12 +11,12 @@ var Hash = bitcore.crypto.Hash;
  * `getPayload` method to modify the message payload.
  * @param {Object=} options
  * @param {String=} options.command
- * @param {Number=} options.magicNumber
+ * @param {Network=} options.network
  * @constructor
  */
 function Message(options) {
   this.command = options.command;
-  this.magicNumber = options.magicNumber;
+  this.network = options.network || Networks.defaultNetwork;
```

Comment:
The default should always be set in Messages factory, I don't think it needs to be set here if it's not defined.